### PR TITLE
Update protect-your-global-administrator-accounts.md

### PR DESCRIPTION
**Pull request by michaelmsonne**
Added Best practices for Azure AD roles

### CODE REVIEW

File: microsoft-365/enterprise/protect-your-global-administrator-accounts.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ Microsoft provides capabilities to help protect your organization, but they're e
 
 3. Protect privileged accounts with Zero Trust identity and device access recommendations.
 
+> [!NOTE]
+> To secure your privileged roles, check out [Best practices for Azure AD roles](/azure/active-directory/roles/best-practices) to securing privileged access to your tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/protect-your-global-administrator-accounts.md]:
```suggestion
> To secure your privileged roles, check out [Best practices for Azure AD roles](/azure/active-directory/roles/best-practices) to secure privileged access to your tenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @michaelmsonne, kindly commit the suggested change. Many thanks.

File: microsoft-365/enterprise/protect-your-global-administrator-accounts.md (hunk 1 above)
Comment (michaelmsonne) [reply microsoft-365/enterprise/protect-your-global-administrator-accounts.md]: Sure, i see 👍

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**